### PR TITLE
fix(sorts) option to ignore accent while sorting the numeric text 

### PR DESCRIPTION
**Pull request by mcallegario**
@ghiscoding 

Identifies a bug in the routine that ignores the accent when sorting the numeric text, in this case it generates an error as shown in the images below:

image

image

image

### CODE REVIEW

File: packages/common/src/services/utilities.ts [hunk 1]
```diff
@@ -883,7 +883,7 @@ export function parseUtcDate(inputDateString: any, useUtc?: boolean): string {
  * @returns
  */
 export function removeAccentFromText(text: string, shouldLowerCase = false) {
-  const normalizedText = text.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
+  const normalizedText = text.normalize?text.normalize('NFD').replace(/[\u0300-\u036f]/g, ''):text;
```
Comment (ghiscoding) [inline packages/common/src/services/utilities.ts:886]: I think it's incorrect syntax isn't it? I think it should always include a dot `?.` instead of `?`

**Review by mcallegario — COMMENTED** [review]
redid the code, see if it's better this way.

File: packages/common/src/services/utilities.ts (hunk 1 above)
Comment (mcallegario) [reply packages/common/src/services/utilities.ts]: redid the code, see if it's better this way.
image

**Comment by ghiscoding** [comment]
yeah that's better thanks

**Pull request merged**